.github/CODEOWNERS

```diff
@@ -62,6 +62,7 @@
 /kubernetes_engine/**/*                @GoogleCloudPlatform/python-samples-reviewers
 /kubernetes_engine/django_tutorial/**/*    @glasnt @GoogleCloudPlatform/python-samples-reviewers
 /language/**/*                         @GoogleCloudPlatform/dee-data-ai @GoogleCloudPlatform/python-samples-reviewers
+/logging/**/*                          @GoogleCloudPlatform/dee-observability @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
Should this be dee-platform-ops instead, in case one of the two members of dee-observability is absent?

Reply:
The expertise of the rest of the team is on GKE and not observability.